app/models/create_expenses_with_receipt.rb

```diff
@@ -0,0 +1,31 @@
+class CreateExpensesWithReceipt
+  attr_reader :expense
+
+  def initialize(expense_params)
+    @params = expense_params
+  end
+
+  def create
+    return false unless picture_exists?
+
+    Expense.transaction do
+      create_expense
+      create_receipt
+    end
+    @expense.persisted?
+  end
```

Comment:
@zamith: This fails when some attribute other than picture is missing. For example, if user is missing: "Failure/Error: expect(creator.create).to be false
ActiveRecord::RecordInvalid: Validation failed: User can't be blank"